spqlios/arithmetic/vec_znx_arithmetic.h

```diff
@@ -34,27 +34,54 @@ typedef struct cnv_pvec_l_t CNV_PVEC_L;
 /** @brief opaque type that represents a prepared right convolution vector product */
 typedef struct cnv_pvec_r_t CNV_PVEC_R;
 
-/** @brief allocates a prepared matrix (release with free) */
-EXPORT VMP_PMAT* vmp_pmat_alloc(const MODULE* module,           // N
-                                uint64_t nrows, uint64_t ncols  // dimensions
-);
+EXPORT void spqlios_free(void* address);
```

Comment:
these two: spqlios_alloc and spqlios_free should be declared in `commons.h` and implemented in `commons.c`

They replace the block lines 50-58 in commons.h that was just a workaround:  the implem of spqlios_alloc takes care of the logic windows/vs/linux.

basically: 
* Linux/Release uses aligned_alloc/free
* Windows/Release uses _aligned_malloc/_aligned_free
* Debug uses spqlios_debug_alloc/spqlios_debug_free

and outside of this function, `aligned_alloc` is forbidden, since it won't work uniformly on all systems.